internal/cli/args.go

```diff
@@ -5,28 +5,35 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"os"
 	"strings"
 	"time"
 
 	"github.com/kballard/go-shellquote"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/kubectl/pkg/cmd/util/editor"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/tilt-dev/tilt/internal/analytics"
 	engineanalytics "github.com/tilt-dev/tilt/internal/engine/analytics"
+	"github.com/tilt-dev/tilt/internal/sliceutils"
 	"github.com/tilt-dev/tilt/pkg/apis/core/v1alpha1"
+	"github.com/tilt-dev/tilt/pkg/logger"
 	"github.com/tilt-dev/tilt/pkg/model"
 )
 
 type argsCmd struct {
-	clear bool
+	streams genericclioptions.IOStreams
+	clear   bool
 }
 
 func newArgsCmd() *argsCmd {
-	return &argsCmd{}
+	return &argsCmd{
+		streams: genericclioptions.IOStreams{Out: os.Stdout, ErrOut: os.Stderr, In: os.Stdin},
```

Comment:
looking over the code... many of these commands do exactly this.. making the pattern consistent.   
observations:
* it seems like a small amount of repeating code which isn't necessary
* it seems like a decision that should only be made once... the decision of what defines the IOStreams.

It would be much better to have the `cli.go` define this once and pass to all the `newXYZCmd`.   

In this case, I recommend a 

```
func newArgsCmd(streams genericclioptions.IOStreams) *argsCmd {
	return &argsCmd{
		streams: streams,
	}
}
```

Reply:
I should add that this could help with testing as well... as the test could check an Out which is a buffer defined by the Test and no directed to stdio